### PR TITLE
Use canvas-prebuilt instead of canvas in order to drop external dependencies

### PR DESCRIPTION
**Pull request by Munter**

### CODE REVIEW

File: lib/spriteBackgroundImages.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 const queryString = require('querystring');
 const { promisify } = require('util');
 const packers = require('./packers');
-let Canvas;
+const canvasPrebuilt = require('canvas-prebuilt');
 
-try {
-    Canvas = require('canvas');
-} catch (e) {}
+const Canvas = canvasPrebuilt.Canvas;
```
Comment (papandreou) [inline lib/spriteBackgroundImages.js]: Could use destructuring?

Comment (Munter) [reply lib/spriteBackgroundImages.js]: Sure. It's that ok with our node version support?

Comment (papandreou) [reply lib/spriteBackgroundImages.js]: Destructuring assignment is node 6.5.0+ so yeah. We should also lebab/prettierify the code a bit :)

Comment (papandreou) [reply lib/spriteBackgroundImages.js]: Getting ready to introduce prettier etc. over here: https://github.com/assetgraph/assetgraph-sprite/pull/182

**Pull request merged**